src/App.css

```diff
@@ -1,38 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Space+Mono:wght@700&display=swap');
```

Comment:
To use the font with styled components, you could do something like this and then remove the css file https://medium.com/@zmommaerts/implementing-google-fonts-into-your-react-project-using-styled-components-25e7b80de02d